app/frontend/styles/components/upload-documents.scss

```diff
@@ -1,7 +1,43 @@
-.upload-group .govuk-button {
+#file-upload {
   margin-bottom: 0;
 }
 
 .hidden-input {
   display: none;
 }
+
+.upload-progress {
+  display: inline-block;
+  position: relative;
+  width: 18px;
+  height: 18px;
+}
+.upload-progress div {
+  box-sizing: border-box;
+  display: block;
+  position: absolute;
+  width: 18px;
+  height: 18px;
+  margin: 0px;
+  border: 3px solid green;
+  border-radius: 50%;
+  animation: lds-ring 1.2s cubic-bezier(0.5, 0, 0.5, 1) infinite;
+  border-color: green transparent transparent transparent;
+}
+.upload-progress div:nth-child(1) {
+  animation-delay: -0.45s;
+}
+.upload-progress div:nth-child(2) {
+  animation-delay: -0.3s;
+}
+.upload-progress div:nth-child(3) {
+  animation-delay: -0.15s;
+}
+@keyframes lds-ring {
```

Comment:
What's lds-ring? (Also at line 24)

Reply:
Well, I know what a ring is. What's lds?

Reply:
@david-mears-dfe I think that's another way/short form of referring to the [loader/spinner.](https://github.com/loadingio/css-spinner/blob/master/dist/ring.html)
Not sure.

Reply:
It was just the name given to the animation (which was pulled from [here](https://loading.io/css/)). But this is going to change to a version of [MalcolmVonMoJ comment](https://github.com/alphagov/govuk-design-system-backlog/issues/28) as result of [TEVA-506](https://dfedigital.atlassian.net/secure/RapidBoard.jspa?rapidView=21&projectKey=TEVA&modal=detail&selectedIssue=TEVA-506).